### DISCUSSION TITLE
五月更新 - 预

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#1115] [通知]
大家在商店的好评给了我敲代码的动力，非常感谢大家对哔哩的喜爱（不过还是低调使用哟，我们圈地自萌就好）。五月的更新有着非常多的新功能，现在公开预览版以供大家尝鲜试用～

## 高亮功能

- 支持漫游（低调使用，服务器自己找）
- 弹幕引擎升级为 [寒霜弹幕使](https://github.com/cotaku/DanmakuFrostMaster)，与官方app的弹幕体验一致
- 支持综合动态
- 支持新动态通知推送

欢迎在 [Github Discussions](https://github.com/Richasy/Bili.Uwp/discussions/1114) 中给自己最喜欢的功能投票（就图一乐）

## 🚀 功能

- 支持洗脑循环 @Richasy (#1111)
- 为评论/消息/空间视频添加相对时间显示 @Richasy (#1110)
- 支持关注列表分组 @Richasy (#1104)
- 在一些弹出面板中显示滚动条 @Richasy (#1102)
- 支持播放界面长按以倍速播放 @Richasy (#1100)
- 强化首页固定功能 @Richasy (#1098)
- 支持PGC收藏夹状态（想看/在看/看过）切换 @Richasy (#1095)
- 支持视频动态通知 @Richasy (#1093)
- 支持协议启动 @Richasy (#1091)
- 支持显示综合动态 @Richasy (#1089)
- 支持字幕简繁互转 @Richasy (#1084)
- 支持番剧漫游 @Richasy (#1078)
- 升级弹幕引擎为寒霜弹幕使 @Richasy (#1075)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2205.0.0'>五月更新 - 预</a>.</em>

**Comment by xyl23333** [comment]
来喽

**Comment by big-feather** [comment]
拉取文字动态的评论好像有些问题
![image](https://user-images.githubusercontent.com/49027292/164950566-1a95a126-d574-4f22-bc5f-bd42abc86579.png)

**Reply by Richasy** [reply]
#1119 修了，下次更新

**Comment by YangSuGuo** [comment]
不在中。。。

**Comment by liukenny961** [comment]
加油！这正是我见过的最好用的UWP之一

**Comment by HAKURELING** [comment]
哔哩漫游提示PGC内容请求失败。。（但是能够正常搜索到港澳台内容）
![image](https://user-images.githubusercontent.com/63252182/164953567-d7d5d490-8634-41b3-83da-13f141dc2966.png)

**Reply by Richasy** [reply]
现在公开的服务绝大部分使用的是旧的 Web Api，你可能会发现在移动端能用的服务在哔哩中无法使用。出现这种情况是因为你所使用的服务没有对上面的 Api 进行反代。所以要么请求服务端对该 Api 提供支持，要么你可以参考 [自定义代理服务器](https://github.com/ipcjs/bilibili-helper/blob/user.js/packages/unblock-area-limit/README.md#%E8%87%AA%E5%AE%9A%E4%B9%89%E4%BB%A3%E7%90%86%E6%9C%8D%E5%8A%A1%E5%99%A8) 自己建立一个简单的代理服务器。

**Comment by cangming99** [comment]
请问有搜索记录功能吗，好像没看到

**Reply by Richasy** [reply]
没有，暂时不打算支持

**Comment by Alphayyds** [comment]
1、正常视窗按快捷键变小窗正常，但是小窗ctrl+m就回不了正常窗口
2、滚动条拖动和触屏左滑右滑调进度很卡，触屏左右滑动时手势幅度稍大了些不够灵敏
3、播放窗口右边的“关联视频”和“评论”希望可以触屏左右滑切换
4、触屏调节音量希望可以在设置中加入开关（经常误触）
5、触屏长按倍速有时候就没有用了

**Reply by Richasy** [reply]
那个……有新的功能需求还是去Issue里提，而且一个Issue对应一个功能需求。

**Reply by Alphayyds** [reply]
好的好的，才报道不懂

**Comment by FengLiuI** [comment]
新版的弹幕字体好像不能设置有无描边还是阴影了，只能是描边

**Comment by Excalibur111302** [comment]
能不能把預覽版的更新提示關掉...
